unfoldingWord/translationCore
# Fix - #2132 & #2297 -  Project Card column line-up and text wrapping

#### This pull request addresses:

This fixes two issues which are fixed by making the project card tabular content be in HTML tables so every card has the same size column for time, book and language (https://github.com/unfoldingWord-dev/translationCore/issues/2297), and also when the text does wrap, it stays under the text rather than the glyph (https://github.com/unfoldingWord-dev/translationCore/issues/2132)

#### How to test this pull request:

- Add multiple projects, look at the projects page, note that the columns all line up to easily look for a certain project. 
- Add projects with a long book title or language (or just resize the window) and the text should wrap, being indented (not under the glyph)
- Run unit test "ProjectCard"


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3060)
